docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+
+FROM osrf/ros:humble-desktop-jammy
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    python3-pip \
+    wget
+
+RUN echo "source /opt/ros/humble/setup.bash" >> /etc/bash.bashrc
+
+WORKDIR /workspace
+
+RUN mkdir -p /workspace/src
+
+RUN wget https://raw.githubusercontent.com/knzo25/CalibrationTools/feature/new_api_documentation/calibration_tools_standalone.repos --directory-prefix=/workspace
```

Comment:
knzo25 to tier4

Reply:
Thanks. Addressed in [2f241c2](https://github.com/tier4/CalibrationTools/pull/164/commits/2f241c2c7bedc921782f72973fbe3eb223239864)

Note: I am assuming this PR will be merged soon-ish, so I used the main branch, so it will not work in the meantime